source/en/skins-system.rst

```diff
@@ -0,0 +1,163 @@
+   The :samp:`.png` extension can be omitted.
+
+   The server will return a :samp:`404` status response in case when textures aren't found.
+
+.. function:: /textures/{nickname}
+
+   By this URL you can get textures for the **nickname** specified in the request. The result is a JSON document
+   of the following format:
+
+   .. code-block:: javascript
+
+      {
+        "SKIN": {
+          "url": "http://example.com/skin.png",
+          "metadata": {
+            "model": "slim"
+          }
+        },
+        "CAPE": {
+          "url": "http://example.com/cape.png"
+        }
+      }
+
+   Depending on available textures for the player, there may be no :samp:`SKIN` or :samp:`CAPE` fields.
+   If the skin model isn't :samp:`slim`, the :samp:`metadata` field will also be omitted.
+
+   The server will return an empty response with :samp:`204` status in case when textures aren't found.
+
+.. function:: /textures/signed/{nickname}
+
+   This request is used in our `server skins system plugin <http://ely.by/server-skins-system>`_ to load textures with
+   the original Mojang's signature. The textures received in the reply can be transferred to the unmodified game client
+   without any changes. The answer will also include additional property with :samp:`name` equal to **ely**.
+
+   .. code-block:: javascript
+
+      {
+        "id": "ffc8fdc95824509e8a57c99b940fb996",
+        "name": "ErickSkrauch",
+        "properties": [
+          {
+            "name": "textures",
+            "signature": "QH+1rlQJYk8tW+8WlSJnzxZZUL5RIkeOO33dq84cgNoxwCkzL95Zy5pbPMFhoiMXXablqXeqyNRZDQa+OewgDBSZxm0BmkNmwdTLzCPHgnlNYhwbO4sirg3hKjCZ82ORZ2q7VP2NQIwNvc3befiCakhDlMWUuhjxe7p/HKNtmKA7a/JjzmzwW7BWMv8b88ZaQaMaAc7puFQcu2E54G2Zk2kyv3T1Bm7bV4m7ymbL8McOmQc6Ph7C95/EyqIK1a5gRBUHPEFIEj0I06YKTHsCRFU1U/hJpk98xXHzHuULJobpajqYXuVJ8QEVgF8k8dn9VkS8BMbXcjzfbb6JJ36v7YIV6Rlt75wwTk2wr3C3P0ij55y0iXth1HjwcEKsg54n83d9w8yQbkUCiTpMbOqxTEOOS7G2O0ZDBJDXAKQ4n5qCiCXKZ4febv4+dWVQtgfZHnpGJUD3KdduDKslMePnECOXMjGSAOQou//yze2EkL2rBpJtAAiOtvBlm/aWnDZpij5cQk+pWmeHWZIf0LSSlsYRUWRDk/VKBvUTEAO9fqOxWqmSgQRUY2Ea56u0ZsBb4vEa1UY6mlJj3+PNZaWu5aP2E9Unh0DIawV96eW8eFQgenlNXHMmXd4aOra4sz2eeOnY53JnJP+eVE4cB1hlq8RA2mnwTtcy3lahzZonOWc=",
+            "value": "eyJ0aW1lc3RhbXAiOjE0ODYzMzcyNTQ4NzIsInByb2ZpbGVJZCI6ImM0ZjFlNTZmNjFkMTQwYTc4YzMyOGQ5MTY2ZWVmOWU3IiwicHJvZmlsZU5hbWUiOiJXaHlZb3VSZWFkVGhpcyIsInRleHR1cmVzIjp7IlNLSU4iOnsidXJsIjoiaHR0cDovL3RleHR1cmVzLm1pbmVjcmFmdC5uZXQvdGV4dHVyZS83Mzk1NmE4ZTY0ZWU2ZDhlYzY1NmFkYmI0NDA0ZjhlYmZmMzQxMWIwY2I5MGIzMWNiNDc2ZWNiOTk2ZDNiOCJ9fX0="
+          },
+          {
+            "name": "ely",
+            "value": "but why are you asking?"
+          }
+        ]
+      }
+
+   By default textures proxying isn't used for this query. To enable it, add an additional GET parameter
+   :samp:`?proxy=true`.
+
+   The server will return an empty response with :samp:`204` status in case when textures aren't found.
```

Comment:
```suggestion
   The server will return an empty response with :samp:`204` status, if textures aren't found.
```
Again, maybe 404 status? I have no idea.

Reply:
No, this is `204` status because all new Mojang's APIs use the same logic.